integrated_channels/degreed2/client.py

```diff
@@ -275,6 +277,36 @@ def create_content_metadata(self, serialized_data):
                 f'Degreed2APIClient create_content_metadata failed with status {status_code}: {response_body}',
                 status_code=status_code
             )
+        # once course is created successfully, we need to do 2 more steps
+        # 1. Fetch skills from enterprise-catalog
+        client = EnterpriseCatalogApiClient()
+        metadata = client.get_content_metadata_content_identifier(
+            enterprise_uuid=self.enterprise_configuration.enterprise_customer.uuid,
+            content_id=external_id
+        )
+        LOGGER.info(
+            generate_formatted_log(
+                self.enterprise_configuration.channel_code(),
+                self.enterprise_configuration.enterprise_customer.uuid,
+                None,
+                None,
+                f'[Degreed2Client] metadata: {metadata}'
+            )
+        )
```

Comment:
This log may not be providing us any useful info that isn't already being logged.

Reply:
yeah you're right! I just added it temporarily to see the exact response in production once to test things. if everything works as expected then I'll remove this log.